core/services/keystore/csa.go

```diff
@@ -181,8 +181,6 @@ func (ks *csa) EnsureKey(ctx context.Context) error {
 		return err
 	}
 
-	ks.logger.Infof("Created CSA key with ID %s", key.ID())
```

Comment:
I suspect that we need to preserve this effect... :thinking: 
Perhaps we can pass a logging hook which only accesses the `ID()` method